src/SignalR/clients/csharp/Client/test/FunctionalTests/HubConnectionTests.cs

```diff
@@ -357,6 +357,51 @@ await foreach (var streamValue in asyncEnumerable)
             }
         }
 
+        [Theory]
+        [MemberData(nameof(HubProtocolsAndTransportsAndHubPaths))]
+        [LogLevel(LogLevel.Trace)]
+        public async Task IAsyncEnumerableServerParam(string protocolName, HttpTransportType transportType, string path)
+        {
+            var protocol = HubProtocols[protocolName];
+            using (StartServer<Startup>(out var server))
+            {
+                var connection = CreateHubConnection(server.Url, path, transportType, protocol, LoggerFactory);
+                try
+                {
+                    async IAsyncEnumerable<int> clientStreamData()
+                    {
+                        for (var i = 0; i < 5; i++)
+                        {
+                            yield return i;
+                            await Task.Delay(10);
+                        }
+                    }
+                    var stream = clientStreamData();
+
+                    await connection.StartAsync().OrTimeout();
+                    var expectedValue = 0;
+                    var streamTo = 5;
+                    var asyncEnumerable = connection.StreamAsyncCore<int>("StreamIAsyncConsumer", new object[] { stream });
```

Comment:
Why isn’t this calling StreamAsync?

Reply:
It can. Was using this API in another test and copied it